src/coreclr/jit/valuenum.cpp

```diff
@@ -8737,6 +8737,10 @@ void Compiler::fgValueNumberTree(GenTree* tree)
         {
             fgValueNumberCastTree(tree);
         }
+        else if (tree->OperGet() == GT_BITCAST)
+        {
+            fgValueNumberBitCast(tree);
```

Comment:
This call should be added under the following `switch` instead:

https://github.com/dotnet/runtime/blob/6b766cde5a05efe008b869913730e8265caa9746/src/coreclr/jit/valuenum.cpp#L8832-L8837

You will also need to update this list of opers to include `GT_BITCAST`:

https://github.com/dotnet/runtime/blob/6b766cde5a05efe008b869913730e8265caa9746/src/coreclr/jit/valuenum.cpp#L6791-L6803

Reply:
Fixed.